### PR TITLE
Update to support silverstripe 5

### PR DESCRIPTION
**Pull request by satrun77**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ tree data as JSON format for use in custom frontend implementation.
```
Comment (TheSceneman) [inline README.md:5]: On line 3, this can be reworded to be more explicit as to what this module is for. Instead of `provides an option to output the entire tree...` can we reword this to `adds PHP functions to the block to output the entire tree...`

File: README.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ in that case, add here and remove the docs folder. You might use this as a quick
 mhave multiple documentation pages.
 
 ## Example configuration (optional)
```
Comment (TheSceneman) [inline README.md]: This heading is a stub from the module template. This heading and content should be deleted from readme. Same goes for documentation heading at line 20

File: composer.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "silverstripeltd/silverstripe-elemental-decisiontree-json",
-    "description": "",
+    "description": "Provide an option to output the entire tree data as JSON format.",
```
Comment (TheSceneman) [inline composer.json]: As per above comment lets reword this to tell devs exactly what this module does

**Review by TheSceneman — CHANGES_REQUESTED** [review]
Can we please re-word module description and remove some irrelevant stubs form the readme

File: README.md (hunk 2 above)
Comment (satrun77) [reply README.md]: LOL. I did not see that, thanks

**Review by TheSceneman — APPROVED** [review]
Nice!
